package/CHANGELOG

```diff
@@ -174,6 +174,8 @@ Enhancements
     checking if it can be used in parallel analysis. (Issue #2996, PR #2950)
 
 Changes
+  * `bat.BAT` now uses the `analysis.base.Results` class to store the `bat`
+    results attribute (Issue #3261, #3272)
```

Comment:
Technically not needed because bat wasn't released – but I think I said in the past "if it has an issue then list it"... so leave it in

Reply:
I changed the issue text to include CHANGELOG entry